<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeVolumePropertiesWidget</class>
 <widget class="QWidget" name="ChangeVolumePropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeVolumePropertiesWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QSlider" name="lowerThresholdSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lowerThresholdDisplay">
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="upperThresholdDisplay">
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QSlider" name="opacitySlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <strikeout>false</strikeout>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string> Thresholds</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Opacity</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QSlider" name="upperThresholdSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>upperThresholdSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ChangeVolumePropertiesWidget</receiver>
   <slot>changeUpperThreshold(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacitySlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ChangeVolumePropertiesWidget</receiver>
   <slot>changeOpacity(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowerThresholdSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ChangeVolumePropertiesWidget</receiver>
   <slot>changeLowerThreshold(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeLowerThreshold(int)</slot>
  <slot>changeUpperThreshold(int)</slot>
  <slot>changeOpacity(int)</slot>
 </slots>
</ui>
